[MTM-63606] Updated docs with info for the new radial gauge. (#3317)
SoftwareAG/c8y-docs: content/cockpit/widgets-collection.md

### Radial Gauge {#radial-gauge}

> The Radial gauge widget visualizes data points in the form of a radial gauge, making it ideal for monitoring values like temperature, pressure, or performance metrics at a glance.

Reviewer: ```suggestion
The "Radial gauge" widget visualizes data points in the form of a radial gauge, making it ideal for monitoring values like temperature, pressure, or performance metrics at a glance.
```

> Presets and Customization

Reviewer: ```suggestion
#### Presets and customization

```

> You can choose from various preset styles such as Default, Pointer, Progress Bar, Progress Indicator, and Grade Rating. Each preset provides a distinct visual design.

Reviewer: ```suggestion
You can choose from various preset styles such as "Default", "Pointer", "Progress bar", "Progress indicator", and "Grade rating". Each preset provides a distinct visual design.
```

![Radial gauge widget](/images/users-guide/cockpit/cockpit-widget-radial-gauge.png)

You must enable at least one data point to create the "Radial gauge" widget.

> For full control, click “Show advanced options” to customize the gauge’s appearance and behavior.

Reviewer: ```suggestion
For full control, click **Show advanced options** to customize the gauge’s appearance and behavior.
```

**Advanced options reference**

| Category         | Property                                             | Description                                           |
| ---------------- | ---------------------------------------------------- | ----------------------------------------------------- |
| **General**      | `name`                                               | Name of the gauge preset                              |

> |                  | `radius`                                             | Radius of the gauge (e.g., "90%")                     |

Reviewer: ```suggestion
|                  | `radius`                                             | Radius of the gauge (for example, "90%")                     |
```

> |                  | `center`                                             | Center position of the gauge (e.g., `["50%", "50%"]`) |

Reviewer: ```suggestion
|                  | `center`                                             | Center position of the gauge (for example, `["50%", "50%"]`) |
```

|                  | `startAngle`, `endAngle`                             | Start and end angles of the gauge arc                 |

> | **Split Lines**  | `splitNumber`                                        | Number of segments in the gauge                       |

Reviewer: ```suggestion
| **Split lines**  | `splitNumber`                                        | Number of segments in the gauge                       |
```

|                  | `splitLineLength`, `splitLineLengthRatio`            | Length of split lines (absolute or ratio)             |
|                  | `splitLineDistance`, `splitLineDistanceRatio`        | Distance of split lines from axis (absolute or ratio) |

> |                  | `splitLineColor`, `splitLineWidth`                   | Color and width of the split lines                    |

Reviewer: ```suggestion
|                  | `splitLineColor`, `splitLineWidth`                   | Color and width of split lines                    |
```

|                  | `tickWidth`, `tickColor`                             | Width and color of ticks                              |
|                  | `tickDistance`, `tickDistanceRatio`                  | Distance of ticks from center (absolute or ratio)     |
|                  | `tickLength`, `tickLengthRatio`                      | Length of ticks (absolute or ratio)                   |
| **Axis**         | `axisLabelDistance`, `axisLabelDistanceRatio`        | Distance of axis labels from center                   |
|                  | `axisLabelColor`                                     | Color of axis labels                                  |
|                  | `axisLabelFontSize`, `axisLabelFontSizeRatio`        | Font size of labels (absolute or ratio)               |
|                  | `axisLabelFontSizeMin`, `axisLabelFontSizeMax`       | Minimum and maximum font size for labels              |
|                  | `axisLineWidth`, `axisLineWidthRatio`                | Width of the axis line (absolute or ratio)            |
| **Pointer**      | `showPointer`                                        | Whether to show the pointer                           |

> |                  | `pointerStyle`, `pointerColor`                       | Style and color of the pointer                        |

Reviewer: ```suggestion
|                  | `pointerStyle`, `pointerColor`                       | Style and color of pointer                        |
```

|                  | `pointerWidth`, `pointerWidthRatio`                  | Width of pointer (absolute or ratio)                  |
|                  | `pointerLength`, `pointerLenghtRatio`                | Length of pointer (absolute or ratio)                 |
|                  | `pointerOffset`                                      | Offset of pointer from center                         |

> | **Progress Bar** | `progressBar`                                        | Enable progress bar                                   |

Reviewer: ```suggestion
| **Progress bar** | `progressBar`                                        | Enable progress bar                                   |
```

### Radial Gauge {#radial-gauge}

The "Radial gauge" widget visualizes data points in the form of a radial gauge, making it ideal for monitoring values like temperature, pressure, or performance metrics at a glance.

#### Presets and customization

You can choose from various preset styles such as "Default", "Pointer", "Progress bar", "Progress indicator", and "Grade rating". Each preset provides a distinct visual design.

> ![Radial gauge widget](/images/users-guide/cockpit/cockpit-widget-radial-gauge.png)

Reviewer: probably the image needs an update as well?

Author: Done

You must enable at least one data point to create the "Radial gauge" widget.

For full control, click **Show advanced options** to customize the gauge’s appearance and behavior.

**Advanced options reference**

| Category         | Property                                             | Description                                           |
| ---------------- | ---------------------------------------------------- | ----------------------------------------------------- |
| **General**      | `name`                                               | Name of the gauge preset                              |

> |                  | `radius`                                             | Radius of the gauge (for example, "90%")               |

Reviewer: ```suggestion
|                  | `radius`                                             | Radius of the gauge (for example, "90%")              |
```

> |                  | `radius`                                             | Radius of the gauge (for example, "90%")                |

Reviewer: ```suggestion
|                  | `radius`                                             | Radius of the gauge (for example, "90%")               |
```

> |                  | `radius`                                             | Radius of the gauge (for example, "90%")                     |

Reviewer: ```suggestion
|                  | `radius`                                             | Radius of the gauge (for example, "90%")                |
```

|                  | `center`                                             | Center position of the gauge (for example, `["50%", "50%"]`) |
|                  | `startAngle`, `endAngle`                             | Start and end angles of the gauge arc                 |
| **Split lines**  | `splitNumber`                                        | Number of segments in the gauge                       |
|                  | `splitLineLength`, `splitLineLengthRatio`            | Length of split lines (absolute or ratio)             |
|                  | `splitLineDistance`, `splitLineDistanceRatio`        | Distance of split lines from axis (absolute or ratio) |

> |                  | `splitLineColor`, `splitLineWidth`                   | Color and width of split lines                    |

Reviewer: ```suggestion
|                  | `splitLineColor`, `splitLineWidth`                   | Color and width of split lines                        |
```

|                  | `unitFontMin`, `unitFontMax`                         | Min and max font size for the unit label              |
|                  | `unitColor`                                          | Color of the unit label                               |
|                  | `dateFontSize`, `dateFontRatio`                      | Font size or ratio of the timestamp                   |
|                  | `dateFontMin`, `dateFontMax`                         | Min and max font size for the timestamp               |
|                  | `dateColor`                                          | Color of the timestamp                                |
| **Details**      | `showDetail`                                         | Show detailed information like value or markers       |
|                  | `valueFontSize`                                      | Font size for the displayed value                     |
|                  | `detailOffsetCenter`                                 | Offset of detail from center                          |
|                  | `showMarkPoint`                                      | Show mark points on the gauge                         |

> 

Reviewer 2: Is there some echarts documentation available where we could link to for more details?

Author: @reey  Yes, but it wouldn't be completely accurate as some of these properties are our own to make a better user experience. E.g. dateFontRatio, unitFontRatio, etc...
